s3/replication/common/tests/system/test_put_object_tagging.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+import os
+from config import Config
+
+
+def main():
+
+    config = Config()
+
+    # CONFIG OPTIONS
+    bucket_name = config.source_bucket_name
+    object_name = config.object_name_prefix
+
+    # Put tagset on the object.
+    command = str(
+        'aws s3api put-object-tagging --bucket ' +
+        bucket_name +
+        ' --key ' +
+        object_name +
+        ' --tagging \'{\"TagSet\" : [{\"Key\":\"user-tag\",'
+        '\"Value\":\"user-tag-value\"}]}\'')
+
+    os.popen(command)
```

Comment:
Exit status can be checked and act accordingly. 
```
  exit_status = os.system(command)

    if exit_status == 0:
        print("\nput-bucket-replication passed! ")
    else:
        os.system('rm -rf temp_policy.json')
        print("\nput-bucket-replication failed! ")
        os._exit(exit_status)
```

Reply:
done